docs/product/custom-dashboards.md

```diff
@@ -24,15 +24,29 @@ Then, choose the metric you wish to aggregate.  For Spans, Embrace supports both
 Finally, add any filters and group-bys.  In addition to our core dimensions, for Spans you can use the Span's name, outcome, duration, and any custom Attributes.
 <img src={require('@site/static/images/Spans widgets > 04 filters and group-bys.png').default} alt="Spans as a Widget option" />
 
+## Combined Timeseries 
 
+You can also create Combined graphs. With Combined type graphs, you can visualize up to 10 timeseries in the same graph and optionally create a custom time series using arithmetic operators.
 
+For example, if you want to create a rate of Crashes in the foreground and Network Connection Errors > 100 ms over your total sessions. To do this, you will:
```

Comment:
This is example can be misleading. Calculating foreground crashes and network connection errors > 100ms sounds like we are looking for sessions that both crashed and also had some network connection error, but this is not currently possible

I like the example that Ocado had for us: they wanted to know the rate of successful network requests but they did not want to include redirects in the total. You can define
A = 2xxs
B = 4xxs
C = 5xxs
F = A / (A + B + C)


I do like emphasizing that combined graphs allow you to visualize multiple sources at the same time, so you can include graphing the volume of foreground crashes and network connection errors > 100ms as an example of that feature.